superbuild/patches/DCMTK_STL_QUIET.patch

```diff
@@ -13,3 +23,19 @@ index 19fb179a6..2512eab89 100755
    CHECK_FUNCTIONWITHHEADER_EXISTS(std::finite "${HEADERS}" HAVE_PROTOTYPE_STD__FINITE)
    CHECK_FUNCTIONWITHHEADER_EXISTS(flock "${HEADERS}" HAVE_PROTOTYPE_FLOCK)
    CHECK_FUNCTIONWITHHEADER_EXISTS(gethostbyname "${HEADERS}" HAVE_PROTOTYPE_GETHOSTBYNAME)
+diff --git a/oflog/libsrc/oflog.cc b/oflog/libsrc/oflog.cc
+index 753c74de4..5f66040ae 100644
+--- a/oflog/libsrc/oflog.cc
++++ b/oflog/libsrc/oflog.cc
+@@ -65,7 +65,7 @@ static void OFLog_init()
+ 
+     console->setLayout(OFmove(layout));
+     rootLogger.addAppender(console);
+-    rootLogger.setLogLevel(dcmtk::log4cplus::INFO_LOG_LEVEL);
++    rootLogger.setLogLevel(dcmtk::log4cplus::FATAL_LOG_LEVEL);
```

Comment:
@Florent2305 it's done, i changed the commented warnings to this setLogLevel method

Reply:
ok